aries-site/src/examples/templates/FilterControls/Filters.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021 - Hewlett Packard Enterprise Company
+
+import { useContext, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Box, Button, Stack, Text } from 'grommet';
+import { Filter } from 'grommet-icons';
+
+import { FilterContext } from '.';
+import { FiltersLayer } from './FiltersLayer';
+
+const FilterButton = styled(Button)`
+  border: 1px solid
+    ${({ theme }) => theme.global.colors.border[theme.dark ? 'dark' : 'light']};
+  &:hover {
+    background: transparent;
+  }
+`;
+
+const ClearFiltersButton = styled(Button)`
```

Comment:
With the recent theme discussions on the above "Filter Button", do you think this styled Button that looks like anchor should be considered as a kind?

Reply:
I'd like to discuss this further. I definitely think it could be a candidate. I'd also like to spend time thinking of any cons associated as well.

I have seen designs which are using anchor styling, but function more like an action button (e.g. "clear filters," and circumstances like help text which opens a side drawer with help content.